include/openthread/thread_ftd.h

```diff
@@ -28,27 +28,61 @@
 
 /**
  * @file
- *   This file includes definitions for a MeshCoP Leader.
+ * @brief
+ *  This file defines the OpenThread Thread API (FTD only).
  */
 
-#ifndef MESHCOP_LEADER_HPP_
-#define MESHCOP_LEADER_HPP_
+#ifndef OPENTHREAD_THREAD_FTD_H_
+#define OPENTHREAD_THREAD_FTD_H_
```

Comment:
The include guard here does not seem to match the filename.
Any particular reason?